src/Open3D/Visualization/Rendering/Filament/TriangleMeshBuffers.cpp

```diff
@@ -110,6 +110,9 @@ VertexBuffer* BuildFilamentVertexBuffer(filament::Engine& engine, const std::uin
                        GetVertexPositionOffset<TexturedVertex>(), stride)
             .normalized(VertexAttribute::TANGENTS)
             .attribute(VertexAttribute::TANGENTS, 0,
+                       VertexBuffer::AttributeType::FLOAT4,
+                       GetVertexTangentOffset<TexturedVertex>(), stride)
+            .attribute(VertexAttribute::CUSTOM0, 0,
```

Comment:
Comment on CUSTOM0?

Reply:
Hm, that's quite strange) I was thinking, that I leaved a comment at least at `FilamentGeometryBuffersBuilder.cpp`, but nothing there too...